src/status_im/navigation/effects.cljs

```diff
@@ -199,15 +199,16 @@
   ([component] (show-overlay component {}))
   ([component opts]
    (navigation/dissmiss-overlay component)
-   (navigation/show-overlay
-    {:component {:name    component
-                 :id      component
-                 :options (merge (options/statusbar-and-navbar-options (:theme opts) nil nil)
-                                 {:layout  {:componentBackgroundColor :transparent
-                                            :orientation              ["portrait"]}
-                                  :overlay {:interceptTouchOutside true
-                                            :handleKeyboardEvents  true}}
-                                 opts)}})))
+   (let [theme (rf/sub [:theme])]
```

Comment:
it's illegal, `(rf/sub [:theme])` can't be used here , subscription can be used only in views, so you have to pass theme as parameter